apps/sms/views/shared/style/sms.css

```diff
@@ -261,3 +261,8 @@ html[dir="rtl"] .contact-header aside.pack-end {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+/* Only change the opacity low storage disabled element */
+.low-storage-disabled {
```

Comment:
I tried to add disable attribute to the button when low storage, but we might still need the button click-able for dialog and overriding the pointer-events css will need !important because the element might be a gaia-component . So I simply apply an reduced opacity here, but the style will be slightly different then real disabled.